CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 * `RealmResults.sort()` and `RealmList.sort()` now return the sorted result instead of sorting in-place.
 * `RealmList.first()` and `RealmList.last()` now throw `ArrayIndexOutOfBoundsException` if `RealmList` is empty.
 * Removed deprecated method `Realm.getTable()` from public API.
+* `Realm.refresh()` and `DynamicRealm.refresh()` on Looper thread now make no effect. `RealmObject` and `RealmResults` are always updated on the next event loop.
```

Comment:
This was partly mentioned in 0.88 as well: "* All thread local change listeners are now delayed until the next Looper event instead of being triggered when committing.", but the semantics changed more in 0.89 so 👍 

A few grammar mistakes: "on a Looper no longer have any effect."

Also `refresh()` will be deprecated by #2386 (not relevant to the changelog though)